subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,29 @@
+from run import run_base
+from dockertest.output import DockerVersion
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def initialize(self):
+        super(run_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(run_tmpfs, self).run_once()
+        try:
+            self.sub_stuff["cont"].wait_by_name(self.sub_stuff['name'])
+        except ValueError:
+            pass  # container already finished and exited
+
+    def postprocess(self):
+        super(run_tmpfs, self).postprocess()
+        actual_exit_status = self.sub_stuff['dkrcmd'].exit_status
+        expected_exit_status = self.config['exit_status']
+        self.failif_ne(actual_exit_status, expected_exit_status,
+                       'actual_exit_status != expected_exit_status')
```

Comment:
Put yourself in the shoes of someone seeing this message. Will this description help them understand the error? How about this one instead?

    'exit status from %s' % self.config['cmd']